src/App.js

```diff
@@ -7,7 +7,11 @@ function App() {
       <header className="App-header">
         <img src={logo} className="App-logo" alt="logo" />
         <p>
-          Edit <code>src/App.js</code> and save to reload.
+          Edit
+          {' '}
+          <code>src/App.js</code>
+          {' '}
+          and save to reload.
```

Comment:
- [ ] Replace the code here with a default line perhaps `<h1>Hello World</h1>` would do for the meantime.